src/configs/split_infer_cifar.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
is it working?

Reply:
Yeah, this is the old CIFAR code. I did run it and saw loss go down (only ran for few epochs though)